libmscore/measure.cpp

```diff
@@ -1440,6 +1443,11 @@ Element* Measure::drop(EditData& data)
                   score()->undoAddElement(e);
                   return e;
 
+            case ElementType::MEASURE_NUMBER:
+                  undoChangeProperty(Pid::MEASURE_NUMBER_MODE, static_cast<int>(MeasureNumberMode::SHOW));
+                  delete e;
+                  break;
```

Comment:
ok, changed `return 0` to `break`, but had to change some other ocurrences of `return 0` for consistency.

Reply:
Hmm, OK. Not sure what to prefer, exit early, or one exit only,

Reply:
It was both, which is in any case not a good option.